src/types/tracked-row.type.ts

```diff
@@ -0,0 +1,4 @@
+export interface TrackedRow {
+  $$index: number;
+  $$expanded: 1 | 0;
```

Comment:
You have to map all rows in order to use expanded property correctly. Since you can do expand all, its required to have a 1-to-1.